src/create-theme-listener.js

```diff
@@ -18,13 +18,20 @@ export default function createThemeListener(CHANNEL = channel) {
 
   function subscribe(context, cb) {
     if (context[CHANNEL]) {
-       return context[CHANNEL].subscribe(cb);
+      return context[CHANNEL].subscribe(cb);
```

Comment:
formatting fixes should always go separate so that we know what has happened in a commit